cmd/start.go

```diff
@@ -42,7 +35,7 @@ func Start(repoName string, tp node.Type) *cobra.Command {
 		Args:         cobra.NoArgs,
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			level, err := logging.LevelFromString(cmd.Flag(loglevel).Value.String())
+			level, err := logging.LevelFromString(cmd.Flag(loglevelFlag.Name).Value.String())
```

Comment:
This code also defaults to INFO by default. To disable this, we should run `logging.LevelFromString` only `if cmd.Flag(loglevelFlag.Name).Value.String() != ""`. Similar to other flags.